tools/individual-pkg-gen/copy-files-removing-deps.ts

```diff
@@ -0,0 +1,201 @@
+import * as path from 'path';
+import * as awsCdkMigration from 'aws-cdk-migration';
+import * as fs from 'fs-extra';
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const lerna_project = require('@lerna/project');
+
+copyFilesRemovingDependencies();
+
+function copyFilesRemovingDependencies(): void {
+  // there is a lerna.json in the individual-packages directory, where this script executes
+  const project = new lerna_project.Project(__dirname);
+  const packages = project.getPackagesSync();
+  const alphaPackages = getAlphaPackages(packages);
+  const jsiiRosettaVersion = project.manifest.devDependencies['jsii-rosetta'];
+  for (const pkg of packages) {
+    if (!packageIsAlpha(pkg)) {
+      continue;
+    }
+
+    const srcDir = pkg.location;
+    const packageUnscopedName = `${pkg.name.substring('@aws-cdk/'.length)}`;
+    const destDir = path.join('.', packageUnscopedName);
+    fs.mkdirpSync(destDir);
+
+    copyOrTransformFiles(pkg, srcDir, destDir);
+  }
+
+  function copyOrTransformFiles(pkg: any, srcDir: string, destDir: string): void {
+    const sourceFiles = fs.readdirSync(srcDir);
+    for (const sourceFileName of sourceFiles) {
+      if (shouldIgnoreFile(sourceFileName)) {
+        continue;
+      }
+
+      const source = path.join(srcDir, sourceFileName);
+      const destination = path.join(destDir, sourceFileName);
+
+      let fileProcessed = false;
+      if (sourceFileName === 'package.json') {
+        const phase1PackageJson = fs.readJsonSync(source);
```

Comment:
Minor: After all the naming changes, `phase1` doesn't mean anything here.

```suggestion
        const packageJson = fs.readJsonSync(source);
```

Reply:
Done.